proposals/break-continue-in-inline-lambdas.md

```diff
@@ -0,0 +1,141 @@
+# Non-local break and continue
+
+* **Type**: Design proposal
+* **Authors**: Alexander Udalov, Nikita Bobko, Pavel Mikhailovskii, Roman Elizarov, Simon Ogorodnik
```

Comment:
The author is the person who had authored the text of the KEEP or otherwise wrote significant portions of it. Others are usually listed as `Contributors`.